plugins/woocommerce-admin/client/products/use-product-helper.ts

```diff
@@ -82,25 +82,23 @@ export function useProductHelper() {
 				status,
 			} ).then(
 				( newProduct ) => {
+					const noticeContent =
```

Comment:
Nitpick, if we move this inside the conditional (`if ( ! skipNotice ) {`) we will set the const `noticeContent` knowing that we will use it, otherwise, we could set it without using it later.

Reply:
Right!!! thanks